incubator/hnc/Makefile

```diff
@@ -198,6 +205,9 @@ ifndef HNC_PAT
 endif
 ifndef HNC_RELEASE_ID
 	$(error HNC_RELEASE_ID is undefined; ${ERR_MSG})
+endif
+ifeq ($(IMG_ALREADY_EXISTS), 0)
```

Comment:
I think this means "the return code from calling gcloud was 0, which means it exists" but the variable makes it sound like a boolean, which has the opposite meaning. I was a bit confused reading this.

Can we either invert the value in this variable, or else give it a more descriptive name (e.g. `IMG_READ_ERROR`)?

Reply:
Absolutely I'll make the change.